### ISSUE TITLE
请教一下epoch和学习率的设置。

### ISSUE THREAD

**Issue by CCzzzzzzz** [hiyouga/ChatGLM-Efficient-Tuning#89]
请教一下这两个参数的设置有什么技巧吗？例如我想在BELLE2M的数据上微调，epoch和learning_rate应该多少合适？

**Comment by hiyouga** [comment]
epoch 和 learning rate 没有先验值，需要根据训练中损失曲线变化情况动态调整。
